docs/locale/fr/LC_MESSAGES/userguide/troubleshooting.po

```diff
@@ -16,133 +16,138 @@ msgstr ""
+msgid ""
+"MS Excel/Word or another MS Office software says the file name is too long"
 msgstr ""
-"MS Excel/Word ou un autre logiciel MS Office indique que le nom du fichier est "
-"trop long"
+"MS Excel/Word ou un autre logiciel MS Office indique que le nom du fichier "
+"est trop long"
 
 #: ../../userguide/troubleshooting.rst:14
 msgid ""
-"This error message occurs when you save or open a file if the path to the file "
-"(including the file name) exceeds 218 characters. This limitation includes "
-"three characters representing the drive, the characters in folder names, the "
-"backslash character between folders, and the characters in the file name."
+"This error message occurs when you save or open a file if the path to the "
+"file (including the file name) exceeds 218 characters. This limitation "
+"includes three characters representing the drive, the characters in folder "
+"names, the backslash character between folders, and the characters in the "
+"file name."
 msgstr ""
-"Ce message d'erreur apparaît lorsque vous enregistrez ou ouvrez un fichier si "
-"le chemin d'accès au fichier (y compris le nom du fichier) dépasse 218 "
-"caractères. Cette limite inclut les trois caractères représentant la lettre du "
-"lecteur, les caractères des noms de dossiers, la barre oblique inverse entre "
-"les dossiers et les caractères du nom du fichier."
+"Ce message d'erreur apparaît lorsque vous enregistrez ou ouvrez un fichier "
+"si le chemin d'accès au fichier (y compris le nom du fichier) dépasse 218 "
+"caractères. Cette limite inclut les trois caractères représentant la lettre "
+"du lecteur, les caractères des noms de dossiers, la barre oblique inverse "
+"entre les dossiers et les caractères du nom du fichier."
 
 #: ../../userguide/troubleshooting.rst:18
 msgid ""
 "`Error message when you open or save a file in microsoft excel filename "
-"<https://support.microsoft.com/en-us/help/213983/error-message-when-you-open-or-"
-"save-a-file-in-microsoft-excel-filename>`_"
+"<https://support.microsoft.com/en-us/help/213983/error-message-when-you-open-"
+"or-save-a-file-in-microsoft-excel-filename>`_"
 msgstr ""
-"`Message d'erreur lors de l'ouverture ou de l'enregistrement d'un fichier dans "
-"microsoft excel nom de fichier <https://support.microsoft.com/en-us/help/213983/"
-"error-message-when-you-open-or-save-a-file-in-microsoft-excel-filename>`_"
+"`Message d'erreur lors de l'ouverture ou de l'enregistrement d'un fichier "
+"dans microsoft excel nom de fichier <https://support.microsoft.com/en-us/"
+"help/213983/error-message-when-you-open-or-save-a-file-in-microsoft-excel-"
+"filename>`_"
 
 #: ../../userguide/troubleshooting.rst:20
 msgid ""
-"So this is a Microsoft issue, and there is nothing Parsec can do about it. The "
-"only solution is to rename the file, or rename one or more folders that contain "
-"the file in order to have shorter names."
+"So this is a Microsoft issue, and there is nothing Parsec can do about it. "
```

Comment:
I think updating MAX_PATH does work. See https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry.
Still a mistery why it's not enabled by default on Windows 11 🤷

Reply:
[`8af7270`](https://github.com/Scille/parsec-cloud/pull/8387/commits/8af72704163b00dc0eb1006ad9ed89e521f1731f)